docs/tools/mbed_targets.md

```diff
@@ -253,6 +257,10 @@ The [`"device_name"`](/docs/development/porting/index.html) attribute it `target
 
 The `device_name` key in `targets.json` is `MK20DX256xxx7` for any target that uses this particular MCU.
 
+#### `detect_code`
+
+The `detect_code` contains four ASCII characters containing only hexadecimal values (A-F and 0-9). This code is the same for all platforms of the same type. For example, all K64F platforms have a detect_code of 0240. `Mbed LS` uses this to identify the platform. This field is not used anymore. `Mbed LS` has its own database of detect codes: https://github.com/ARMmbed/mbed-ls/blob/master/mbed_lstools/platform_database.py#L45
```

Comment:
Could we please use an example other than the K64F?

Reply:
@AnotherButler Is there a reason for that? Just curious!

Reply:
In general, we prefer to use generic examples that don't involve specific boards, but if we need to include a board-specific example, we prefer to showcase all of our Mbed Enabled boards. We've highlighted the K64F in the past, so it's nice to let another board shine.

Reply:
I'd be fine with just dropping the K64F example. Besides, the Mbed LS link has "all of the examples" 😄

Reply:
Actually come to think of it, since we are already providing the link to all of the examples, it might be a better idea to drop the example altogether. Thoughts, @AnotherButler ?

Reply:
@aashishc1988 Sounds good to me 👍